### ISSUE TITLE
Could I use it as a SDK, integrate to my desktop app?

### ISSUE THREAD

**Issue by yuanhoujun** [Genymobile/scrcpy#3200]
I have a desktop app, developed with Flutter, I want to integrate the projection screen function into my app, how should I do it? Can  I use it as a desktop SDK into my project?

**Comment by yuanhoujun** [comment]
??

**Comment by rom1v** [comment]
No, scrcpy uses APIs which require shell permissions, not grantzd to Android apps.

**Comment by rom1v** [comment]
No, there is no "libscrcpy". You could use the scrcpy-server and program your own client though.

**Comment by yuanhoujun** [comment]
OK, that will be perfect, if we can use it in my flutter app.
